### ISSUE TITLE
Unable to open more than 2 webcams

### ISSUE THREAD

**Issue by ousmanou** [sarxos/webcam-capture#139]
@sarxos hi!
 thank for your Webcam capture , but i am unable to open more than 2 webcams
i need some helps
i always got this exception 

``` plain
Exception in thread "pool-2-thread-1" com.github.sarxos.webcam.WebcamException: Cannot execute task
    at com.github.sarxos.webcam.WebcamProcessor$AtomicProcessor.process(WebcamProcessor.java:57)
    at com.github.sarxos.webcam.WebcamProcessor.process(WebcamProcessor.java:120)
    at com.github.sarxos.webcam.WebcamTask.process(WebcamTask.java:35)
    at com.github.sarxos.webcam.ds.cgt.WebcamOpenTask.open(WebcamOpenTask.java:20)
    at com.github.sarxos.webcam.Webcam.open(Webcam.java:187)
    at com.github.sarxos.webcam.Webcam.open(Webcam.java:147)
    at com.github.sarxos.webcam.WebcamPanel.start(WebcamPanel.java:529)
    at cameraGUI$RunCamera.run(cameraGUI.java:147)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:722)
Caused by: com.github.sarxos.webcam.WebcamException: Cannot start native grabber!
    at com.github.sarxos.webcam.ds.buildin.WebcamDefaultDevice.open(WebcamDefaultDevice.java:220)
    at com.github.sarxos.webcam.ds.cgt.WebcamOpenTask.handle(WebcamOpenTask.java:38)
    at com.github.sarxos.webcam.WebcamProcessor$AtomicProcessor.run(WebcamProcessor.java:66)
    ... 3 more
```

**Comment by sarxos** [comment]
Just FYI, the newest [webcam-captrure-driver-v4l4j](https://github.com/sarxos/webcam-capture/tree/master/webcam-capture-drivers/driver-v4l4j) (Linux only) should select the most appropriate (i.e. the most bandwidth-friendly) image format. Therefore, at least on Linux, opening more webcams at once should technically possible.

**Comment by nestormace** [comment]
Dear Sarxos:
I have a similar problem. I'm trying to connect 11  USB IPEVO HD Plus webcams to a PC running a Java program using a USB hub. The PC runs Windows XP.
I'm testing with just 2 webcams. The program open() both cameras, then reads several still images from both cameras and then close() both cameras.
I used to get all black images from the second camera and out of focus images.
Then I added a delay of around 2 seconds between each open() and all other commands and between each close() and any other command. I could get a correct behaviour for resolution of up the 1270 X 780.
Thing get worse with higher resolution - some runs present the old problems.
Things are also worse when using a hub than when using a PC USB port for each camera.
Is there some way to avoid these problems or at least to improve my "solution"?
